README.md

```diff
@@ -12,6 +12,15 @@ To install Docker on your system execute the following command:
 curl https://raw.githubusercontent.com/larics/uav_ros_simulation/main/installation/dependencies/docker.sh | bash
 ```
 
+## Troubleshooting
+
+* Check ```CHANGELOG.md``` for any new changes added to this project
+* ```git pull origin main --rebase``` - in case there are new changes to the challenge repository.
+* ```git pull lmark1/uav_ros_simulation:[[DISTRO]]``` - in case the Docker container simulation is not working correctly.
+* ```./docker_build --build-args "--no-cache --pull" --[[DISTRO]]``` - in case the Docker container simulation is (still) not working correctly.
+
+**NOTE** - If the challenge does not setup correctly it is not your fault! Try following the troubleshooting recommendation. If your problem is not resolved please post an issue.
```

Comment:
If the challenge does not setup correctly it is probably not your fault! :)

Reply:
Added keyword probably :sweat_smile: